### PR TITLE
[Merged by Bors] - refactor(Probability/Kernel/CondCdf): mv some theorems

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Probability/Kernel/CondCdf.lean [hunk 1]
```diff
@@ -6,6 +6,7 @@ Authors: Rémy Degenne
 import Mathlib.MeasureTheory.Measure.Stieltjes
 import Mathlib.MeasureTheory.Decomposition.RadonNikodym
 import Mathlib.MeasureTheory.Constructions.Prod.Basic
+import Mathlib.Logic.Encodable.Basic
```
Comment (mo271) [inline Mathlib/Probability/Kernel/CondCdf.lean:9]:
```suggestion
import Mathlib.Logic.Encodable.Basic
import Mathlib.MeasureTheory.Measure.Stieltjes
import Mathlib.MeasureTheory.Decomposition.RadonNikodym
import Mathlib.MeasureTheory.Constructions.Prod.Basic
```

File: Mathlib/Logic/Encodable/Basic.lean [hunk 2]
```diff
@@ -665,6 +665,16 @@ theorem le_sequence (a : α) : f a ≤ f (hf.sequence f (encode a + 1)) :=
   hf.rel_sequence a
 #align directed.le_sequence Directed.le_sequence
 
+variable (hf : Directed (· ≥ ·) f)
```
Comment (RemyDegenne) [inline Mathlib/Logic/Encodable/Basic.lean:668]: Can you please put sections around sequence_mono and le_sequence on one hand, and around sequence_anti and sequence_le on the other hand, in order to restrict the scope of the `variable` declarations?

By the way, thanks a lot for moving these theorems. I left those post-port tasks and should have done them a while ago.

Comment (mo271) [reply Mathlib/Logic/Encodable/Basic.lean:676]: sections added.

**Comment by RemyDegenne** [comment]
Thanks!
bors r+

**Pull request closed without merging**